eng/pipelines/coreclr/templates/perf-job.yml

```diff
@@ -13,14 +13,16 @@ parameters:
   codeGenType: 'JIT'
   projetFile: ''
   runKind: ''
+  runJobTemplate: ''
```

Comment:
Can we default this to `run-performance-template.yml`? And then just override it for the scenarios job.

Reply:
Second thought .. I'm wondering why do we default everything to microbenchmarks tests? Sometimes if we just take default values we are closer to getting false-positive results. I agree that we should have a default value for common traits like timeout.

Reply:
I was specifically talking about the project file because 90% of the project files are the microbenchmark one. What other default values impact the results?

Reply:
I see, ```runJobTemplate``` param is for choosing which yaml template to set up the payload . Maybe in addition to the template ```projectFile``` in ```run-performance-job.yml``` https://github.com/dotnet/runtime/blob/154e971c2f026963c2cd2e15d78e27601801151e/eng/pipelines/coreclr/templates/run-performance-job.yml#L21 can be set to ```microbenchmarks.proj``` since that's the single proj file for microbenchmarks and we won't have to specify it in the parent template any more.